doc/mgr/orchestrator_cli.rst

```diff
@@ -128,6 +164,7 @@ Decommission an OSD
 Removes an OSD from the cluster and the host, if the OSD is marked as
```

Comment:
ceph orchestrator osd rm <osd-id>,[<osd-id]

Several osd_ids can be provided

Reply:
in rook.rook_cluster.RookCluster#rm_service, only supports one OSD. How do we want to specify this here in the documentation?

Reply:
It would be nice that all the orchestrators support to remove several osd's simultaneously, but if it not is possible, maybe a note in the documentation and an error message from the orchestrator when you try to do something not possible for the orchestrator will be enough.

I think that It does not seem sensible "to cut" functionality because one orchestrator cannot do one type of operation. Not all the operations will be easy/possible to implement in all the orchestrators, if we "cut"  things each time we found one of this cases, probably the final result will be very limited.

Reply:
Fair enough. How do we want to specify this here in the documentation? Maybe a `.. note::`?

Reply:
IMHO. This is going to be something that is going to happen more frequently than we will wish... The orchestrator-cli is just an interface though other orchestrator modules, in fact, other manager modules can decide to use directly and internal orchestrator instead to use the orchestrator-cli.

But the orchestrator-cli is a very good tool to let us to unify criteria and check-things with different orchestrators in an easy way.
What i suggest is  to put the more generic interface in the orchestrator-cli doc, and a general note at the begiinig warning the user that specific requirements/functionality (and in the future even special operations) must be checked in each  specific orchestrator.

Reply:
fixed. Actually Rook doesn't implement osd rm